golem_messages/message.py

```diff
@@ -351,8 +355,9 @@ class Hello(Message):
     ] + Message.__slots__
 
     def __init__(self, **kwargs):
-        self.golem_messages_version = golem_messages.__version__
         super().__init__(**kwargs)
+        if self.golem_messages_version is None:

```

Comment:
Does this mean no version received?
In that case would it be safer to set the version to 0.0.0?

Reply:
It can mean two things: 
 1. either no version received (no slot information or `None` in slot)
 1. a new message created

It could be hard to differentiate between these two cases. Assuming no malicious messages, empty value means older protocol id, and connection will be dropped. Assuming malicious message it doesn't really matter what we put there.

Reply:
After giving it a second thought I found a way to address this issue in a clean way

Reply:
Looks great! thank you for the reply and fix :)